suite-common/wallet-utils/src/discreetModeUtils.ts

```diff
@@ -0,0 +1,30 @@
+import { createContext, useContext } from 'react';
```

Comment:
Not sure if _all of this_ belongs to `wallet-utils`. Seemed suitable to me. Where would you place it?

Note: there are already some react-related functions in this package, so it's not strictly business logic.